dataset_eval/gaia_wrapper.py

```diff
@@ -104,7 +105,7 @@ def train_svm_history(project, params, output_file_path):
     gt.className = "highlevel." + project["className"]
 
     history = train_svm(ds, gt, **params_model)  # doing the whole training
-    history.save(output_file_path)
+    history.save(str(output_file_path))
```

Comment:
the `str`? yes - this is passed into gaia, and the bindings only accept strings as arguments, not unicode. The original code `tempfile.mkdtemp()` returned a string, but now we create something with the job id, which is unicode (from the database). On second thoughts, we should probably use `.encode("utf-8")` here instead.